spring-cloud-stream-binder-kafka/pom.xml

```diff
@@ -58,7 +69,7 @@
 		<dependency>
 			<groupId>org.springframework.integration</groupId>
 			<artifactId>spring-integration-kafka</artifactId>
-			<version>${spring-integration-kafka.version}</version>
+			<version>2.1.1.BUILD-SNAPSHOT</version>
```

Comment:
As I allude to in the PR description; it needs https://github.com/spring-projects/spring-integration-kafka/pull/163